chromadb/api/segment.py

```diff
@@ -135,6 +142,13 @@ def create_collection(
         for segment in segments:
             self._sysdb.create_segment(segment)
 
+        self._telemetry_client.capture(
+            ClientCreateCollectionEvent(
+                collection_uuid=str(id),
+                embedding_function=embedding_function.__class__.__name__,
```

Comment:
is it possible someone gives us an embedding_function that isn't a class and so __class__ would be undefined, throwing an error?

Reply:
It would just return `function` as the name. Functions in python have a `__class__` attribute whose `__name__` is function.  

We could use `embedding_function.__name__` instead but then we just get the variable name, and it might be anonymous.